### PR TITLE
Certificate plugin custom BT#16493

### PR DESCRIPTION
**Pull request by aragonc**
Este es una mejora del plugin de certificado personalizado para cursos de la tarea https://task.beeznest.com/issues/16493

Ojo: Antes se subían este certificado imagenes, sellos y logos, es demasiado no visual, ahora toda la parte del contenido del certificado se coloca en el editor CKEditor, donde por medio de la tabla se coloca el logo y el contenido, incluido las firmas.

Luego el fondo se personaliza como antes solo se sube un fondo en tamaño a4 a 300dpi que cubre toda la hoja, y si quieres agregar un sello, este flota al lado derecho del certificado.

### CODE REVIEW

File: plugin/customcertificate/resources/css/form.css [hunk 1]
```diff
@@ -15,4 +15,16 @@
     margin: 0 10px 0 10px;
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216&bid=15555139)

File: plugin/customcertificate/resources/css/form.css [hunk 2]
```diff
@@ -15,4 +15,16 @@
...
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
+    list-style: none;
+    margin: 0;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 3]
```diff
@@ -15,4 +15,16 @@
...
+    padding: 0;
+}
+.list-tags li{
+    display: inline-block;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 4]
```diff
@@ -15,4 +15,16 @@
...
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+.list-tags li{
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 5]
```diff
@@ -15,4 +15,16 @@
     margin: 0 10px 0 10px;
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 6]
```diff
@@ -15,4 +15,16 @@
     margin: 0 10px 0 10px;
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
+    list-style: none;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 7]
```diff
@@ -15,4 +15,16 @@
...
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 8]
```diff
@@ -15,4 +15,16 @@
     margin: 0 10px 0 10px;
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 9]
```diff
@@ -15,4 +15,16 @@
...
+
+.list-tags{
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+.list-tags li{
+    display: inline-block;
+    margin-right: 10px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 10]
```diff
@@ -15,4 +15,16 @@
...
+    margin-top: 5px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 7 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/src/index.php [hunk 11]
```diff
@@ -275,36 +270,44 @@
...
+}
+$strInfo.='</ul>';
+$createCertificate = '<strong>'.get_lang('CreateCertificateWithTags').'</strong>';
 $form->addElement(
     'html',
-    Display::return_message($createCertificate.': <br />'.$strInfo, 'normal', false)
+    Display::return_message($createCertificate.': '.$strInfo, 'normal', false)
 );
 $form->addHtml('</div>');
 $form->addHtml('</fieldset>');
```
Comment (AngelFQC) [inline plugin/customcertificate/src/index.php]: En la línea 248 cambias un `fieldset` por un `h4`, el cual se cierra,  pero el `fieldset` no tiene etiqueta de cierre. Esta es la etiqueta de cierre que debería borrarse... tal vez?

También se podría usar `FormValidator::addHeader`

File: plugin/customcertificate/database.php [hunk 12]
```diff
@@ -42,18 +42,8 @@
...
+$certificateTable->addColumn('logo', Type::STRING);
 $certificateTable->addColumn('seal', Type::STRING);
-$certificateTable->addColumn('signature1', Type::STRING);
-$certificateTable->addColumn('signature2', Type::STRING);
-$certificateTable->addColumn('signature3', Type::STRING);
-$certificateTable->addColumn('signature4', Type::STRING);
-$certificateTable->addColumn('signature_text1', Type::STRING);
-$certificateTable->addColumn('signature_text2', Type::STRING);
-$certificateTable->addColumn('signature_text3', Type::STRING);
-$certificateTable->addColumn('signature_text4', Type::STRING);
```
Comment (AngelFQC) [inline plugin/customcertificate/database.php]: ¿Estas columnas no pueden ser opcionales? Asumo que hay gente usando el plugin usando estas columnas.

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 44
- Added 33
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216&bid=15555139)

File: plugin/customcertificate/resources/css/certificate.css [hunk 13]
```diff
@@ -1,11 +1,59 @@
...
+.contents-learnpath td {
+    background: #FFFFFF;
+    font-size: 12px;
+    height: 160mm;
+    vertical-align: top;
+}
+
+ol {
+    margin: 0;
+    padding: 0;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 14]
```diff
@@ -1,11 +1,59 @@
...
+}
+
+.logo-seals {
+    height: 100px;
+    text-align: center;
+    vertical-align: middle;
+}
+
+@page caraA {
+    height: 210mm;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 15]
```diff
@@ -1,11 +1,59 @@
...
+.caraB {
+    box-decoration-break: slice;
+    page: caraB
+}
+
+.contents-learnpath td {
+    background: #FFFFFF;
+    font-size: 12px;
+    height: 160mm;
+    vertical-align: top;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 16]
```diff
@@ -1,11 +1,59 @@
...
+
+.seals {
+    font-size: 16px;
+    text-align: center;
+}
+
+.logo-seals {
+    height: 100px;
+    text-align: center;
+    vertical-align: middle;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 17]
```diff
@@ -1,11 +1,59 @@
...
+}
+
+@page caraA {
+    height: 210mm;
+    margin: 0;
+    padding: 0;
+    page-break-before: always;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 18]
```diff
@@ -1,11 +1,59 @@
...
+
+.caraB {
+    box-decoration-break: slice;
+    page: caraB
+}
+
+.contents-learnpath td {
+    background: #FFFFFF;
+    font-size: 12px;
+    height: 160mm;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 19]
```diff
@@ -3,16 +3,36 @@
...
+.list-tags {
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list-tags li {
+    display: inline-block;
+    margin-right: 10px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 20]
```diff
@@ -1,11 +1,59 @@
...
+    font-size: 18px;
+}
+
+.logo {
+    height: 150px;
+    vertical-align: middle;
+}
+
+.seals {
+    font-size: 16px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 21]
```diff
@@ -1,11 +1,59 @@
...
+    background: #FFFFFF;
+}
+
+.caraB {
+    box-decoration-break: slice;
+    page: caraB
+}
+
+.contents-learnpath td {
+    background: #FFFFFF;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 22]
```diff
@@ -3,16 +3,36 @@
...
+    background: #eee;
+}
+
+label {
+    font-size: 13px;
+}
+
+.list-tags {
+    display: block;
+    list-style: none;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 23]
```diff
@@ -1,11 +1,59 @@
...
-.seals { font-size: 16px; text-align:center;}
-.logo-seals { height: 100px; text-align:center; vertical-align: middle;}
-@page caraA { height: 210mm; margin:0; padding:0; page-break-before:always;} 
-.caraA {box-decoration-break: slice; page: caraA}
-@page caraB { background:#FFFFFF; }
-.caraB {box-decoration-break: slice; page: caraB}
-.contents-learnpath td {background: #FFFFFF; font-size: 12px; height: 160mm; vertical-align: top;}
-ol {margin: 0; padding:0;}
+body {
+    background-repeat: no-repeat;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 24]
```diff
@@ -3,16 +3,36 @@
...
+    margin: 0;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 25]
```diff
@@ -3,16 +3,36 @@
...
+
 .certificado-text-label {
     font-weight: 700;
     margin: 0 10px 0 10px;
 }
-input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+
+input:disabled {
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 21 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 26]
```diff
@@ -1,11 +1,59 @@
...
+@page caraA {
+    height: 210mm;
+    margin: 0;
+    padding: 0;
+    page-break-before: always;
+}
+
+.caraA {
+    box-decoration-break: slice;
+    page: caraA
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 27]
```diff
@@ -1,11 +1,59 @@
...
+}
+
+.caraB {
+    box-decoration-break: slice;
+    page: caraB
+}
+
+.contents-learnpath td {
+    background: #FFFFFF;
+    font-size: 12px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 28]
```diff
@@ -3,16 +3,36 @@
...
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list-tags li {
+    display: inline-block;
+    margin-right: 10px;
+    margin-top: 5px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 29]
```diff
@@ -1,11 +1,59 @@
...
+    box-decoration-break: slice;
+    page: caraA
+}
+
+@page caraB {
+    background: #FFFFFF;
+}
+
+.caraB {
+    box-decoration-break: slice;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 30]
```diff
@@ -1,11 +1,59 @@
...
+    page: caraB
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 31]
```diff
@@ -3,16 +3,36 @@
...
+
+input:disabled {
+    background: #eee;
+}
+
+label {
+    font-size: 13px;
+}
+
+.list-tags {
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 26 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 21 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 32]
```diff
@@ -3,16 +3,36 @@
...
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list-tags li {
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 33]
```diff
@@ -1,11 +1,59 @@
...
+}
+
+.seals {
+    font-size: 16px;
+    text-align: center;
+}
+
+.logo-seals {
+    height: 100px;
+    text-align: center;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 34]
```diff
@@ -3,16 +3,36 @@
...
 }
-input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+
+input:disabled {
+    background: #eee;
+}
+
+label {
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 44
- Added 33
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216&bid=15555139)

File: plugin/customcertificate/resources/css/certificate.css (hunk 33 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css [hunk 35]
```diff
@@ -3,16 +3,36 @@
...
+
+.list-tags {
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list-tags li {
+    display: inline-block;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 17 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 21 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 36]
```diff
@@ -1,11 +1,59 @@
...
+    vertical-align: middle;
+}
+
+.seals {
+    font-size: 16px;
+    text-align: center;
+}
+
+.logo-seals {
+    height: 100px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 21 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 32 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 13 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 26 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 25 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 29 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 20 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 37]
```diff
@@ -1,11 +1,59 @@
...
-ol {margin: 0; padding:0;}
+body {
+    background-repeat: no-repeat;
+    margin: 0;
+    padding: 0;
+}
+
+td {
+    font-family: arial, sans-serif;
+    font-size: 18px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 30 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 16 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 31 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 24 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 26 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 23 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 38]
```diff
@@ -1,11 +1,59 @@
...
+
+.contents-learnpath td {
+    background: #FFFFFF;
+    font-size: 12px;
+    height: 160mm;
+    vertical-align: top;
+}
+
+ol {
+    margin: 0;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css (hunk 30 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 39]
```diff
@@ -1,11 +1,59 @@
...
+    padding: 0;
+}
+
+td {
+    font-family: arial, sans-serif;
+    font-size: 18px;
+}
+
+.logo {
+    height: 150px;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/certificate.css [hunk 40]
```diff
@@ -1,11 +1,59 @@
...
-.contents-learnpath td {background: #FFFFFF; font-size: 12px; height: 160mm; vertical-align: top;}
-ol {margin: 0; padding:0;}
+body {
+    background-repeat: no-repeat;
+    margin: 0;
+    padding: 0;
+}
+
+td {
+    font-family: arial, sans-serif;
```
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/certificate.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 34 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

File: plugin/customcertificate/resources/css/form.css (hunk 22 above)
Comment (jmontoyaa) [inline plugin/customcertificate/resources/css/form.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)

**Comment by ywarnier** [comment]
@aragonc en el estado actual de este PR (no respondes a las observaciones de @AngelFQC ni corriges los espacios reportados por codeclimate y tampoco pones bien los espacios entre "if" y la paréntesis que sigue) es muy probable que lo rechace.
Podrías darle una revisión (en particular con la observación de Angel sobre los campos signature) antes de que iniciemos el trabajo de corregir todos los últimos detalles?

File: plugin/customcertificate/src/index.php (hunk 11 above)
Comment (ywarnier) [reply plugin/customcertificate/src/index.php:303]: Parece que fue corregido en 4bd56152f22ac90d507bf181ef0f7d4316706600

**Pull request closed without merging**